src/main/resources/db/migration/changes/RecommendationRequest-02.json

```diff
@@ -0,0 +1,31 @@
+{
+    "databaseChangeLog": [
+      {
+        "changeSet": {
+          "id": "RecommendationRequest-2",
```

Comment:
Can we make the id consistent with the filename?

And while we are at it, let's make them both
```suggestion
          "id": "RecommendationRequest-002",
```

I doubt we'll have more than 99 migrations on this database table over the course of the lifetime of this app, but I'm even more confident that we won't have more than 999.